### DISCUSSION TITLE
Sample preview

### DISCUSSION THREAD

**Discussion by egasimus** [bangcorrupt/hacktribe#181] [Ideas]
Would it be possible to listen to samples from the SD card when selecting a sample to import in the Import Sample menu?

**Comment by bangcorrupt** [comment]
I'm not sure.  If I can get sample loading via sysex working then I'll try.

**Reply by maks** [reply]
Hacktribe is already a game changer for me using my E2. Sample loading via sysex, even if there was a size limit would be a HUGE game changer on top of that! 🙂🤞
